apps/torus-worker/src/db.ts

```diff
@@ -50,6 +50,14 @@ export async function insertAgentWeight(weights: AgentWeight[]) {
         percComputedWeight: w.percComputedWeight,
       })),
     )
+    .onConflictDoUpdate({
+      target: [computedAgentWeightSchema.agentKey],
+      set: buildConflictUpdateColumns(computedAgentWeightSchema, [
+        "atBlock",
+        "computedWeight",
+        "percComputedWeight",
+      ]),
+    })
     .execute();
```

Comment:
Question: Should we persist historical data, thus making the unique constraint over the (agentKey AND atBlock) and making the query considering the last `atBlock` or just keep the last weights? @EdSDR 
Otherwise LGMT.